### DISCUSSION TITLE
Any guidelines migrate from react-native-animatable to moti?

### DISCUSSION THREAD

**Discussion by jeremyspritelyco** [nandorojo/moti#191] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

`It't not a bug. It's a question.`

### Versions

```markdown
- Moti: latest
- Reanimated: 2.2.4
- React Native: 0.66.0
```

### Reproduction

🙃

**Comment by nandorojo** [comment]
hmm i don’t have a guide currently, but definitely open to a PR! want to show some examples from animatable and i can show them with moti?
